suite-native/device/src/selectors.ts

```diff
@@ -237,3 +239,9 @@ export const selectIsDeviceSetupSupported = createMemoizedSelector(
         (isDeviceSetupSupported(model) ||
             (isModelTDeviceOnboardingFeatureFlagEnabled && model === DeviceModelInternal.T2T1)),
 );
+
+export const selectShouldDeviceBeTreatedAsBootloaderMode = createMemoizedSelector(
```

Comment:
The name is confusing. The main point here is that it is a device with bootloader mode, that is not completely new and it is probably initialized, right? Would be nice to test it with device that has FW installed but no seed yet.

Reply:
I like the name of the selector, but `const isDeviceInBootloaderMode = useSelector(selectShouldFactoryResetBeVisible);` is confusing on many places. And `const shouldDeviceBeTreatedAsBootloaderMode = useSelector(selectShouldFactoryResetBeVisible);` even more confusing.

Reply:
Good point, I have also renamed the selector uccurances in https://github.com/trezor/trezor-suite/pull/19433/commits/fdccc7c1be7de2c5769c536bcd8b887f561ba37a